src/development/tools/devtools/memory.md

```diff
@@ -12,58 +12,74 @@ managed by the Dart VM (virtual machine).
 
 ## DevTools memory page
 
-DevTools Memory page lets you peek at how an isolate is using
+The DevTools Memory page lets you peek at how an isolate is using
 memory at a given moment. 
 
 Memory profiling in DevTools consists of 3 main functions:
+
 * Charting memory usage statistics and events
-* Anaylsis to view all memory via a heap to detect memory issues and inspect objects
-* Allocations to monitor and track (stack trace) specific classes and objects when an allocation occurs
+* Anaylsis to view all memory via a heap to detect memory issues
+  and inspect objects
+* Allocations to monitor and track (stack trace) specific classes
```

Comment:
Allocation monitoring to track object allocations and their stack traces for selected classes.

Reply:
Done